_includes/org-table.html

```diff
@@ -16,7 +16,7 @@ <h6 class="no-matches" style="display: none;">No matches.</h6>
         <tr>
           <td>
             <a href="https://github.com/{{ org }}" title="{{ org }}">
-            <img src="https://github.com/{{ org }}.png" width="40" height="40" alt="{{ org }}"></a>
+            <img src="https://avatars.githubusercontent.com/{{ org }}?v=3&amp;s=40" width="40" height="40" alt="{{ org }}"></a>
```

Comment:
Let me play around and see if I can figure something out. May be able to do a mod on the org name length or something.